example/app/data/DemoDataSource/plugins/attribute-selector/blueprints/AttributeSelectorSidebarBlueprint.json

```diff
@@ -3,6 +3,11 @@
   "name": "AttributeSelectorSidebarBlueprint",
   "extends": ["CORE:NamedEntity"],
   "attributes": [
+    {
+      "attributeType": "string",
+      "type": "dmss://system/SIMOS/BlueprintAttribute",
+      "name": "type"
+    },
```

Comment:
the attribute "type" should be included in CORE:NamedEntity, so I don't think it is necessary to add the "type" attribute to AttributeSelectorSidebarBlueprint.

Could you double check this?

Reply:
added it to backlog, do it another PR